classes/music_album.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(publish_date, archived, _genre, _author, _label)
+    super(publish_date, archived)
+    @on_spotify = on_spotify
```

Comment:
While `@on_spotify` takes the value of `on_spotify`, there is nowhere a place where `on_spotify` is instantiated or passed to the method as a parameter.